src/sss_client/sss_cli.h

```diff
@@ -442,6 +442,7 @@ enum response_type {
                               * be used together with other prompting options
                               * to determine the type of prompting.
                               * @param None. */
+    SSS_PAM_CERT_INFO_WITH_HINT,
```

Comment:
Maybe adding a "documentation" here (and in SSS_PAM_CERT_INFO), as those are the only ones missing docs?
Or are those trivial enough to do not need any docs?